ra_sender.go

```diff
@@ -51,6 +51,22 @@ func newRASender(initialConfig *InterfaceConfig, ctor rAdvSocketCtor, logger *sl
 	}
 }
 
+func (s *raSender) setOptions(config *InterfaceConfig) []ndp.Option {
+	options := []ndp.Option{
+		&ndp.LinkLayerAddress{
+			Direction: ndp.Source,
+			Addr:      s.sock.hardwareAddr(),
+		},
+	}
+
+	if config.MTU != 0 {
```

Comment:
Nit: MTU can be a negative number since it's an `int`. We already validate it at startup or on Reload, but I'd write it `> 0` for clarity.

Reply:
I fixed